ui-manchette/src/components/helpers.ts

```diff
@@ -55,23 +55,33 @@ export const calcOperationalPointsToDisplay = (
 
 export const operationalPointsHeight = (
   operationalPoints: StyledOperationalPointType[],
+  height: number,
   zoom: number,
   isProportional: boolean
-) =>
-  operationalPoints.map((op, index) => {
+) => {
+  if (operationalPoints.length < 2) return [];
+  const from = operationalPoints[0].position;
+  const to = operationalPoints[operationalPoints.length - 1].position;
```

Comment:
How would do you feel to start using this syntax ? I think it's more concise. And it forces us to check if the value is undefined so maybe it's more safe ?

```suggestion
  const to = operationalPoints.at(-1)!.position;
```

Reply:
Yes you're right, it might be better :+1: